sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_auth_async.py

```diff
@@ -74,7 +75,7 @@ async def test_client_sas_credential_async(live_eventhub):
         await producer_client.send_batch(batch)
 
     # Finally let's do it with SAS token + conn str
-    token_conn_str = "Endpoint=sb://{}/;SharedAccessSignature={};".format(hostname, token.decode())
```

Comment:
Would a customer ever interact with this `token`?

Reply:
when you say interact, do you string manipulation or just moving the whole token string around?

a customer may pass the token string to downstream for distribution, but as far as I know they should not change the token string itself which would make the token invalid.

Reply:
I ask because it looks like `token` has changed from bytes to str - so wondering if that's breaking that I no longer needed to call `token.decode()`

Reply:
yea.. from "bytes to str" perspective, it's a breaking change..

I know our credential `AccessToken` protocol says we should return string, so we were wrong at the beginning.
I could update the code to return bytes instead of string in EH to make it non-breaking.

Reply:
Yeah if this is customer facing we'll probably have to do that.